.circleci/config.yml

```diff
@@ -242,8 +242,6 @@ workflows:
           matrix:
             parameters:
               version:
-                - v2.0.0
-                - v2.3.9
-                - v2.4.0
-                - v2.5.0-alpha.4
```

Comment:
Probably unrelated: Should the version numbers for the API perf tests be in the various version branches instead of in a matrix against master?

Reply:
Maybe? I kinda like defining what runs all in one place since this is a nightly-only run. We could have each version branch have it's own trigger in circle-ci and maybe that makes more sense organizationally and maintenance wise. I don't have a strong opinion, so I'm happy to switch it if you think it's better, but I think it's a bigger change than I should add to this PR. I'll make a ticket to circle back and re-evaluate.